### ISSUE TITLE
ACLs with Action's using tags where no nodes with that tag don't parse.

### ISSUE THREAD

**Issue by kyhwana** [juanfont/headscale#390]
ACLs won't parse if you have an ACL where there are no nodes with the tag (or namespace?) specified in the ACL
For example:
``` 
 "tagOwners": {
            "tag:externalservers": [ "group:admin" ] 
        },
[snip]
{
            "Action": "accept",
            "Users": [ "group:users", "group:admin" ],
            "Ports": [ "externalservers:22" ]
        }
```

- `Version 0.15-beta3
- 1.22.0

```
Mar  4 02:22:23 headscaletest headscale[471972]: 2022-03-04T02:22:23Z DBG Loading ACL policy from path func=LoadACLPolicy path=/etc/headscale/acls.json
Mar  4 02:22:23 headscaletest headscale[471972]: 2022-03-04T02:22:23Z ERR Error parsing ACL 8, Port 0
Mar  4 02:22:23 headscaletest headscale[471972]: 2022-03-04T02:22:23Z ERR Could not load the ACL policy error="invalid user section" path=/etc/headscale/acls.json                   
````

As in, there are no nodes with the "externalservers" tag. If this is the case, the ACL file will fail to parse.
Removing the Action ACL and restarting headscale results in the ACL json file being parsed correctly.

**Comment by restanrm** [comment]
I believe this bug should have been fixed with  #384

**Comment by restanrm** [comment]
Also, the tag will not be matched in the port section here. You should use the full name ie `tag:externalservers`

**Comment by restanrm** [comment]
@kyhwana did you check if this solved your issue ?

**Comment by kyhwana** [comment]
Looks like this was fixed in #384
